create_s3background_account_cipher.py

```diff
@@ -0,0 +1,201 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+#!/usr/bin/env python3
+
+from shutil import register_unpack_format
+import ldap
+import sys
+from s3backgrounddelete.cortx_s3_config import CORTXS3Config
+import ldap.modlist as modlist
+import traceback
+from s3backgrounddelete.cortx_cluster_config import CipherInvalidToken
+from s3backgrounddelete.cortx_s3_cipher import CortxS3Cipher
+
+LDAP_USER = "cn=sgiamadmin,dc=seagate,dc=com"
+LDAP_PASSWORD = "ldapadmin"
+LDAP_URL = "ldapi:///"
+
+g_default_input_params = {
```

Comment:
Write one-liner comment on data and functions

Reply:
Done.